client/components/RepositoryCheck.jsx

```diff
@@ -13,7 +13,8 @@ const INFO_TEXT = {
               </p>,
   commitmessage: <p>The commit message feature (<a href='https://zappr.readthedocs.org/en/latest/setup/#commitmessages'>docs</a>) will check that commit messages
                     in a pull request match at least one of some patterns you provide.
-                 </p>
+                 </p>,
+  pullrequest: <p>The pull request check will verify that pull request's title and body match required length or content checks</p>
```

Comment:
Can we name the new check `specification` or something similiar? That's what we are actually checking for :)